what_to_watch/suggest/suggestion.py

```diff
@@ -14,4 +22,18 @@ def suggest_movie() -> str:
     ---
     str : suggestion
     '''
-    return "Space Jam"
+    name = 'space jam'
+
+    if vals[0] == 'movie':
```

Comment:
Probably good to define this as a constant, maybe an enum that can be accessed elsewhere as well? Maybe `const.py` as an Enum from the standard enum package?